package.json

```diff
@@ -16,6 +16,7 @@
     "start": "node scripts/custom-config start",
     "build": "node scripts/custom-config build",
     "deploy": "gh-pages -d build",
+    "lint": "eslint src/",
```

Comment:
i think you should add a regex for only .js files cause this will even lint the scss files and may throw errors